docs/overview/script-storage.md

```diff
@@ -42,7 +42,7 @@ To find who is using Office Scripts with the search tool, add `.osts` in the **F
 
 ## See also
 
-- [Sharing Office Scripts in Excel for the Web](https://support.microsoft.com/office/sharing-office-scripts-in-excel-for-the-web-226eddbc-3a44-4540-acfe-fccda3d1122b)
+- [Sharing Office Scripts in Excel for the web](https://support.microsoft.com/office/sharing-office-scripts-in-excel-for-the-web-226eddbc-3a44-4540-acfe-fccda3d1122b)
```

Comment:
This is matching the casing on the linked article. I think we should leave it as is.

```suggestion
- [Sharing Office Scripts in Excel for the Web](https://support.microsoft.com/office/sharing-office-scripts-in-excel-for-the-web-226eddbc-3a44-4540-acfe-fccda3d1122b)
```